### DISCUSSION TITLE
Does Apache SuperSet provides integration with AWS Timestream Database

### DISCUSSION THREAD

**Discussion by hitesh1610** [apache/superset#19704] [Q&A / Help]
Does Apache SuperSet provides integration with AWS Timestream Database ?

As per Apache SuperSet documentation, only Amazon Athena and Amazon Redshift are only supported.

AWS Timestream is TimeSeries Database, below link

https://aws.amazon.com/timestream/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc&amazon-timestream-blogs.sort-by=item.additionalFields.createdDate&amazon-timestream-blogs.sort-order=desc

**Comment by srinify** [comment]
Since this isn't a bug, moving to Discussions! Note for the future, this has been discussed / answered in the Superset Slack!

**Reply by IntuzMalvinL** [reply]
Does Apache SuperSet provides integration with AWS Timestream Database #19704

**Reply by i-mann** [reply]
Considering Slack's extremely poor search capability, surely it would be appropriate to share either the link to the conversation, or the result(s) of the conversation here?
